crates/pathfinder/src/storage/schema/revision_0009.rs

```diff
@@ -0,0 +1,104 @@
+use crate::storage::schema::PostMigrationAction;
+use anyhow::Context;
+use rusqlite::Transaction;
+
+pub(crate) fn migrate(transaction: &Transaction) -> anyhow::Result<PostMigrationAction> {
+    // Add new columns to the blocks table
+    transaction
+        .execute_batch(
+            r"ALTER TABLE starknet_blocks ADD COLUMN gas_price INTEGER NOT NULL
```

Comment:
```suggestion
            r"ALTER TABLE starknet_blocks ADD COLUMN gas_price BLOB NOT NULL
```
Don't think `INTEGER` supports 128 bits.

Reply:
Late night copy paste error I think, as I temporarily had the gas price in `u64`, thanks for spotting.